app/scripts/metamask-controller.js

```diff
@@ -5014,8 +5028,12 @@ export default class MetamaskController extends EventEmitter {
             console.log(e);
           }
         },
+        getCaveat: this.permissionController.getCaveat.bind(
+          this.permissionController,
+        ),
         getCurrentChainId: () =>
           this.networkController.state.providerConfig.chainId,
+        getChainPermissionsFeatureFlag: () => process.env.CHAIN_PERMISSIONS,
```

Comment:
Isn't this a string?
```suggestion
        getChainPermissionsFeatureFlag: () => process.env.CHAIN_PERMISSIONS === 'true' ? true : false,
```

Reply:
usually I think people make it equal `1`? I tend to set it for the build like `CHAIN_PERMISSIONS=1 yarn start`. Leaving it as is makes it agnostic to what its set to and only checks that its defined

Reply:
I suppose it could be
```suggestion
        getChainPermissionsFeatureFlag: () => !!process.env.CHAIN_PERMISSIONS,
```

Reply:
In that case, I think this is more idiomatic these days:
```suggestion
        getChainPermissionsFeatureFlag: () => Boolean(process.env.CHAIN_PERMISSIONS),
```